checker/manual/fenum-checker.tex

```diff
@@ -123,6 +121,16 @@ \section{Running the Fenum Checker\label{fenum-running}}
         \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
 \end{alltt}
 
+or, provide the a fully qualified path to the root directory of your project, containing your
+annotations, using the \code{-AqualDirs} option, and also adding the directory to the bootclass path, for example:
+
+\begin{alltt}
+  javac -Xbootclasspath/p:\textit{/full/path/to/myproject:/full/path/to/otherLib} \ttbs
+  		-processor org.checkerframework.checker.fenum.FenumChecker \ttbs
+        \textit{-AqualDirs=/full/path/to/myproject:/full/path/to/otherLib} MyFile.java ...
+\end{alltt}
+
+Note that in this example, \<qual.MyFenum> would exist in either the myproject folder or the otherLib folder.
```

Comment:
Please place "myproject" and "otherLib" in tt italic font, consistent with their use a few lines above.